gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -1170,12 +1208,19 @@ def main(args):
 
             ht = validate_hts[data_type]
             if for_joint:
-                ht = ht.annotate(
-                    info=ht.info.annotate(
-                        **validate_hts["exomes"][ht.key].info,
-                        **validate_hts["genomes"][ht.key].info,
+                in_joint_ht = set(ht.info.keys())
+                for dt in ["exomes", "genomes"]:
+                    info_expr = validate_hts[dt][ht.key].info
+                    info_expr = info_expr.select(
+                        *[f for f in info_expr if f not in in_joint_ht]
+                    )
+                    ht = ht.annotate(info=ht.info.annotate(**info_expr))
+                    ht = ht.annotate_globals(
+                        **{
+                            f"{dt}_{f}": validate_hts[dt].globals[f]
+                            for f in validate_hts[dt].globals
+                        }
```

Comment:
This is already inside a datatype for loop and globals have the prefix
```suggestion
                    ht = ht.annotate_globals(**validate_hts[dt].index_globals())
```

Reply:
It was under testing, now should work. Keep forgetting `index_globals()`.